docs/admin/install/helm-install/google-gke.md

```diff
@@ -3,6 +3,186 @@ tags:
+    
+3.  Install kubectl using `gcloud components install kubectl` command
+    
+4.  Create cluster using a command such as the following example:
+
+    ```  
+    gcloud container clusters create CLUSTER_NAME --num-nodes 2 --machine-type e2-highcpu-8 --zone ZONE_NAME
+    ```
+    where `CLUSTER_NAME` is the name you choose for the cluster and `ZONE_NAME` is the name of Availability Zone, for example: us-west1-a (https://cloud.google.com/compute/docs/regions-zones/) where the cluster resources live in.
+
+5.  Install [Helm3](https://helm.sh/docs/intro/install/)    
+
+6.  Create `jans` namespace where our resources will reside
+    ```
+    kubectl create namespace jans
+    ```
+
+## Jans Installation using Helm
+1.  Install [Nginx-Ingress](https://github.com/kubernetes/ingress-nginx), if you are not using Istio ingress
+    
+      ```
+      helm repo add ingress-nginx https://kubernetes.github.io/ingress-nginx
+      helm repo add stable https://charts.helm.sh/stable
+      helm repo update
+      helm install nginx ingress-nginx/ingress-nginx
+      ```
+
+2.  Create override.yaml file and add changes as per your desired configuration:
+
+    - FQDN/domain not registered:
+    
+        
+        Get the Loadbalancer IP: 
+        ```
+        kubectl get svc nginx-ingress-nginx-controller --output jsonpath='{.status.loadBalancer.ingress[0].ip}'
+        ```
+
+      
+        
+        Add this to override.yaml:
+
+        ```yaml
+        global:
+            lbIp: #Add IP from previous command
+            isFqdnRegistered: false
+        ```
+
+    - FQDN/domain registered:
+    
+        If the FQDN for jans i.e `demoexample.jans.org` is registered and globally resolvable, you have to map it to the loadbalancer IP created in the previous step by Nginx-Ingress.
+        More details in this [guide](https://medium.com/@kungusamuel90/custom-domain-name-mapping-for-k8s-on-gcp-4dc263b2dabe).
+
+        Add this to override.yaml:
+
+        ```yaml
+        global:
+            lbIp: "" #Add LoadBalancer IP from previous command
+            isFqdnRegistered: true
+            fqdn: demoexample.jans.org #CHANGE-THIS to the FQDN used for Jans
+        nginx:
+          ingress:
+              enabled: true
+              path: /
+              hosts:
+              - demoexample.jans.org #CHANGE-THIS to the FQDN used for Jans
+              tls:
+              - secretName: tls-certificate
+                hosts:
+                - demoexample.jans.org #CHANGE-THIS to the FQDN used for Jans
+        ```
+
+
+
+
+
+
+    -  LDAP/Opendj for persitence storage
+
+
+          Add this to override.yaml:
+          ```yaml
+          global:
+            cnPersistenceType: ldap
+            storageClass:
+              provisioner: kubernetes.io/gce-pd
+            opendj:
+              enabled: true
+          ```
+
+          So now configuring both LDAP and no-FQDN will look something like that:
```

Comment:
Either you include the full override.yaml below or you specifically mention the `global` section because some people have the tendency to copy the snippet as the final and try to execute with it. I'd put the full override.yaml below